runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -56,6 +64,16 @@ J9::ClassEnv::convertClassOffsetToClassPtr(TR_OpaqueClassBlock *clazzOffset)
    return (J9Class*)((TR_OpaqueClassBlock *)clazzOffset);
    }
 
+TR_OpaqueClassBlock *
+J9::ClassEnv::convertClassPtrToClassOffset(J9Class *clazzPtr)
```

Comment:
I am suggesting to make this similar to J9::ClassEnv::convertClassOffsetToClassPtr which uses a cast without going through J9JitMemory (which does the same thing).